hnn_core/tests/test_drives.py

```diff
@@ -116,6 +116,30 @@ def test_drive_seeds(setup_net):
     assert ~np.any(np.allclose(trial1_spikes, trial2_spikes))
 
 
+def test_clear_drives(setup_net):
+    """Test clearing drives updates Network"""
+    net = setup_net
+    weights_ampa = {'L5_pyramidal': 0.3}
+    synaptic_delays = {'L5_pyramidal': 1.}
+
+    n_gids = net._n_gids
+    net.add_evoked_drive(
```

Comment:
Just wanted to make sure we are testing removing multiple drives. Does the setup_net already have defined drives?

Reply:
Just updated the test to explicitly add two drives